modules/nw-anp-np-reference.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/logging-network-security.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-anp-differences-networkpolicy_{context}"]
+= Key differences between AdminNetworkPolicy and NetworkPolicy custom resources
+
+The following table explains key differences between the cluster scoped `AdminNetworkPolicy` API and the namespace scoped `NetworkPolicy` API.
+
+[cols="1,1,1"]
+|===
+|Policy elements | AdminNetworkPolicy | NetworkPolicy
+
+|Applicable user
+|Cluster administrator or equivalent
+|Namespace owners
+
+|Scope
+|Cluster
+|Namespaced
+
+|Drop traffic
+|Supported with an explicit `Deny` action set as a rule.
+|Supported via implicit `Deny` isolation at policy creation time.
+
+|Delegate traffic
+|Supported with an `Pass` action set as a rule.
+|Not applicable
+
+|Allow traffic
+|Supported with an explicit `Allow` action set as a rule.
+|The default action for all rules is to allow.
+
+|Rule precedence within the policy
+|Depends on the order in which they appear within an ANP. The higher the rule's position the higher the precedence.
+|Rules are additive
+
+|Policy precedence
+|Among ANPs the `priority` field sets the order for evaluation. The lower the priority number higher the policy precedence.
+|There is no policy ordering between policies.
+
+|Feature precedence
+|Evaluated first via tier 1 ACL and BANP is evaluated last via tier 3 ACL.
+|Enforced after ANP and before BANP, they are evaluated in tier 2 of the ACL.
+
+|Matching pod selection
+|Can apply different rules across namespaces.
+|Can apply different rules across pods in single namespace.
+
+|Cluster egress traffic
+|Supported via `nodes` and `networks` peers
+|Supported through `ipBlock` field along with accepted CIDR syntax.
+
+|Cluster ingress traffic
+|Not supported
+|Not supported
+
+|Fully qualified domain names (FQDN) peer support
+|Not supported
+|Not supported
+
+|Namespace selectors
+|Supports advanced selection of Namespaces with the use of `namespaces.matchLabels` field
+|Supports label based namespace selection with the use of `namespaceSelector` field
+
```

Comment:
@JoeAldinger Really like this table.
@tssurya Would it make sense to add a row about peer evaluation in ANP and BANP?  Unless it may open up can of worms.

Reply:
let's do it at a later stage if need peer? for now let's try to get the crux in